src/App.java

```diff
@@ -14,17 +14,126 @@ public void start() {
         printInstructions();
         doMadlib();
     }
-    
+
     /**
-     * Document what this method does here!
+     * This intruction tells you what you are supposed to do.
```

Comment:
In the future, please make sure to put comments like this - starting with the /** symbol - above all methods you create. These methods serve as a special signal to VS Code and other editors that they are documenting method functionality, which will be very useful in future projects.